cmd/evm/main.go

```diff
@@ -233,6 +233,10 @@ func (ruleSet) IsAtlantis(*big.Int) bool {
 	return true
 }
 
+func (ruleSet) IsAgharta(*big.Int) bool {
+	return true
```

Comment:
this should be a blocknumber?

Reply:
Nah ruleSet always defaults to true for tests, the functional one is a different type. iirc but not at computer now